pro/src/screens/Bookings/BookingsRecapTable/components/Filters/__specs__/FilterByBookingStatus.spec.tsx

```diff
@@ -107,10 +107,10 @@ describe('components | FilterByBookingStatus', () => {
       // then
       const filterIcon = screen.getByRole('img')
       expect(filterIcon).toHaveAttribute(
-        'src',
-        expect.stringContaining('ico-filter-status-red.svg')
+        'class',
+        expect.stringContaining('active')
```

Comment:
On a le même test que plus haut pour tester 2 choses différentes. Ça me parait pas bon.

Reply:
Au dessus c'est un .not() normalement, pour vérifier que l'icône est bien noir